examples/django_demo/generic_foreignkey/migrations/0001_initial.py

```diff
@@ -21,7 +21,13 @@ class Migration(migrations.Migration):
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('tag', models.SlugField()),
                 ('object_id', models.PositiveIntegerField()),
-                ('content_type', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='contenttypes.ContentType')),
+                (
+                    'content_type',
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        to='contenttypes.ContentType'
+                    )
+                ),
```

Comment:
Migrations are autogenerated. It would be a good idea to exclude them from formatting checks (I realize now that the isort PR contained a migration file too).

Reply:
Why would migrations be treated specifically? Yes, some can be automatically generated, but they should follow the same standards as the rest of the code base. Manually written migrations would benefit from `flake8` linting, even though the examples are fake projects and I doubt there’ll ever be data migrations. Similarly, sorting imports helps developers reading and maintaining the code.

Reply:
My remark was probably better suited to code that is recreated by the machine constantly, and not subject to manual adjustments. In these cases, having to run the linter each time after the machine has done its work to reformat it seems more like a hassle with no clear benefit. This is not the case with migrations, as they are only generated once and may be edited manually at will.